src/pyodide/internal/workers.py

```diff
@@ -607,6 +640,60 @@ async def text(self) -> str:
         return await self.js_object.text()
 
 
+def to_python(obj: "JsProxy", throw=True):
```

Comment:
Maybe let's just call these `to_js` and `to_py`?

Reply:
I think we'll need to give them different names since they are a bit specialised for the serialisation. For example we don't allow converting tuples.